### ISSUE TITLE
Issues running __examples__/building-from-scratch

### ISSUE THREAD

**Issue by shanehandley** [amzn/sketch-constructor#5]
Hi,

Firstly, great job with this library, I've had a lot of fun playing around with it.

I've noticed that when running the example `building-from-scratch` the outputted sketch file contains the correct groups / text values, but the swatches are all black. Additionally when reading the outputted sketch file, the JSON values for colours all appear as R: 0, G:0, B: 0.

It looks as though each colour passes through `TinyColor` in multiple iterations, for which the final results is `{ alpha: 1, blue: 0, green: 0, red: 0 }`. I've never seen `TinyColor` so stopped there.

Cheers

**Comment by dbanksdesign** [comment]
Someone using this, awesome! Thank you so much for raising the issue. 

This happened when I was doing some refactoring and forgot to update the example code. Take a look at this PR for the fix: #6

Basically I added a `Rectangle` class to use instead of `ShapeGroup.Rectangle` Hope that makes sense. Let me know if you come across any more bugs or have any suggestions, this is a side project and very rough right now.

**Comment by aharvard** [comment]
Hi @dbanksdesign, having fun playing around with this library as well! 

I'm having similar issues with the color swatches rendering black in **Sketch 54.1**. I noticed what you did in #6 and it seems that I pulled down the latest. 

For my `Swatch.js` I have
```
const swatchColor = new Rectangle({
      width: 200,
      height: 100,
      x: 0,
      y: 0,
      name: args.value,
      style: {
        fills: [
          {
            color: args.value,
          },
        ],
      },
    });
```

In addition to this occurring in the example `building-from-scratch` it's also happening with `generating-sketch-from-style-dictionary` — thought I would post on this closed issue first before opening a new one.

I'm incredibly new to this sort of thing but I noticed that the sketch API declares a [fillType](https://developer.sketch.com/reference/api/#stylefilltype) on [fills](https://developer.sketch.com/reference/api/#fill) — tried to make it work myself and failed. I don't really know what I'm doing.
```
shape.style.fills = [
  {
    color: '#c0ffee',
    fill: Style.FillType.Color,
  },
]
```

Are you able to reproduce black fills for the swatches that are generated?

Thanks!
